cmd/kubeadm/app/features/features.go

```diff
@@ -39,7 +39,7 @@ const (
 	// StoreCertsInSecrets is alpha in v1.8 and v1.9
 	StoreCertsInSecrets = "StoreCertsInSecrets"
 
-	// DynamicKubeletConfig is alpha in v1.9
+	// DynamicKubeletConfig is beta in v1.10
```

Comment:
This is incorrect.
Loading Kubelet config from a file is beta, along with the kubeletconfig API group.
Dynamic Kubelet config is not yet beta.

Reply:
@mtaufen - do you have a tracking issue?  There was an original parent issue that was closed and folks had assumed it was all in for the freeze.

Reply:
https://github.com/kubernetes/features/issues/281

Reply:
Thanks! Let's hold until DKC graduates to beta in 1.11 as per the feature targets in https://github.com/kubernetes/features/issues/281.